addons/html_builder/static/src/plugins/vertical_alignment_option_plugin.js

```diff
@@ -5,14 +5,13 @@ import { VerticalAlignmentOption } from "@html_builder/plugins/vertical_alignmen
 import { withSequence } from "@html_editor/utils/resource";
 import { VERTICAL_ALIGNMENT } from "@html_builder/utils/option_sequence";
 
-class VerticalAlignmentOptionPlugin extends Plugin {
+export class VerticalAlignmentOptionPlugin extends Plugin {
     static id = "verticalAlignmentOption";
     resources = {
         builder_options: [
             withSequence(VERTICAL_ALIGNMENT, {
                 OptionComponent: VerticalAlignmentOption,
-                selector:
-                    ".s_text_image, .s_image_text, .s_three_columns, .s_showcase, .s_numbers, .s_faq_collapse, .s_references, .s_accordion_image, .s_shape_image",
+                selector: this.selector,
```

Comment:
We could also achieve the same result by using the patch_builder_options resource and giving it a name. Don't know which is better in this case.